docs/comments.txt

```diff
@@ -0,0 +1,813 @@
+  E        4     6     24       float variable	-9.22337e+18 - 9.22337e+18
+  D        8     2     16       double variable	-1.70141e+38 - 1.70141e+38
+  ----------------------------------------------------------------
+                       122 bytes per star
+
+
+
+SUMMARY OF THE AGASC version 1.8 ENTRIES
+
+ Each of the FITS regions files in the AGASC1.8 will contain the
+ following fields for each entry:
+
+
+BYTES NAME - brief description
+
+4    AGASC_ID - a unique long integer used for identification.
+    Currently a binary-packing of the region number, Hubble GSC star
+    number, and Tycho Output Catalog identifier TYC3.
+    No default value (must have an entry).
+
+8    RA - double variable expressing right ascension in decimal degrees.
+    No default value (must have an entry).
+
+8    DEC - double variable expressing declination in decimal degrees.
+    No default value (must have an entry).
+
+2    POS_ERR - short integer value of position uncertainty, in milli-arcsec.
+    Default value of -9999 indicates no error available, or POS_ERR>32767.
+
+1    POS_CATID - unsigned integer identifying the source of the
+    ra, dec, and pos_err.  Default value is 0.
+        0 - no associated catalog
+        1 - GSC1.1
+        2 - PPM
+        3 - Tycho Output Catalog (Tycho-1)
+        4 - ACT
+        5 - Tycho-2
+        6 - GSC-ACT
+        7 - Gaia DR3
+
+4    EPOCH - float variable identifying the epoch of the ra and dec
+    measurements. Default value of -9999.0
+
+2    PM_RA - short integer variable expressing proper motion in ra in units of
+    milli-arcsec per year.     Default value of -9999.
+
+2    PM_DEC - short integer variable expressing proper motion in dec in units
+    of milli-arcsec per year.    Default value of -9999.
+
+1    PM_CATID - unsigned integer identifying the source of the
+    pm_ra and pm_dec.  The codes are the same as listed for pos_catid.
+    Default value is 0.
+
+2    PLX - short integer variable expressing parallax in units of
+    milli-arcsec.    Default value of -9999.
+
+2    PLX_ERR - short integer variable expressing parallax error
+    in units of milli-arcsec.    Default value of -9999.
+
+1    PLX_CATID - unsigned integer identifying the source of the
+    pm_ra and pm_dec.  The codes are the same as listed for pos_catid.
+    Default value is 0.
+
+4    MAG_ACA - float variable expressing the calculated magnitude in the AXAF
+    ACA bandpass in units of magnitude. There is no default value.
+
+2    MAG_ACA_ERR - short integer expressing the uncertainty of mag_aca in
+    units of 0.01mag. There is no default value.
+
+2    CLASS - short integer code identifying classification of entry.
+     Default value of 0.
+        0 - star
+        1 - galaxy
+        2 - blend or member of incorrectly resolved blend.
+        3 - non-star
+        5 - potential artifact
+        6 - known multiple system
+        7 - close to galaxy or other extended object
+
```

Comment:
I note the general agasc description rst now talks about the class=100+ cases, but that doesn't apply to the fits files.

Reply:
I hear you about documentation for *us*.  But the previously existing system, and the one that is actually documented in the catalog-description.rst, points to having a readme.txt at top level and a comments.txt file in the tables directory.  If you want to write a script to make those files from catalog-description.rst that's one option.  Another option was just to make them manually, which I did here with the idea that really nobody uses the catalog except us anyway.  A third option would be to update the catalog-description.rst to remove the references to readme.txt and comments.txt and replace the readme with a text file that just says "See https://sot.github.io/agasc/catalog-description.html" .

Reply:
Oh, and catalog-description-1p7.rst was needed so that both could actually be available from our documentation at https://sot.github.io/agasc/ .  At the time, both agasc versions were actually in use.  If there's a driver to remove the previous documentation we can talk about that as well.